scripts/modify_json.py

```diff
@@ -68,23 +68,44 @@ def modify_json(input_folder, output_folder, variable_name, variable_value):
     variable_value: Float
         The value of the variable we want to create
     """
-    # if the output_folder does not exist it is created via code
-    if not os.path.exists(output_folder):
-        os.mkdir(output_folder)
-
-    jsons = os.listdir(input_folder)
-
-    for jfile in jsons:
-        jfile_path = os.path.join(input_folder, jfile)
-        jdict = json_to_dict(jfile_path)
-        jdict[variable_name] = variable_value
-        save_path = os.path.join(output_folder, jfile)
-        save_dict(jdict, save_path)
+    try:
+        jsons = os.listdir(input_folder)
+    except FileNotFoundError:
+        print('Input file cannot be found or does not exist')
+        sys.exit(0)
```

Comment:
the exit code of `0` means that the script/program exited without problem.  in this case, i would think an invalid input folder is an error, so `1` would be more appropriate.  in the context of this script, its not a big deal, more of a fyi.